### DISCUSSION TITLE
Security Onion Post Installation

### DISCUSSION THREAD

**Discussion by techn00** [Security-Onion-Solutions/securityonion#7350] [Unsupported Versions] [answered]
I have successfully installed SO. Port mirroring, logging and netflow yet to configure on network devices.
Do you have to configure anything on Security Onion before kicking on the network traffic ?

**Comment by tomjones1977** [comment, marked as answer]
Port mirroring should be set up when you configure the monitoring and management interfaces (assuming you have a standalone installation).  The internal tools such as Stenographer, Zeek etc will run automatically (assuming again that you have configured these during setup).

Logging will depend on what logs you are trying to ingest.  For plain syslog you just need to add the source IP ranges to the firewall, Filebeat is already set up to receive them.  See https://docs.securityonion.net/en/2.3/syslog.html

I would recommend installing the Filebeat agent on your servers if you can, it parses and processes logs into ELK more effectively than plain Syslog does.  https://docs.securityonion.net/en/2.3/beats.html

For Netflow the setup is a bit more complex as you need to reconfigure the way Filebeat runs on the SO server.  There is a guide in the documentation for this but if you are new to SO I would recommend getting used to network and log data before trying to change the SO configurations.  https://docs.securityonion.net/en/2.3/filebeat.html?highlight=netflow#walkthrough-netflow-logs
